invisible_cities/evm/event_model.py

```diff
@@ -292,18 +292,21 @@ def __repr__(self):
         return self.__str__()
 
 
-class Blob(VoxelCollection):
-    """A Blob is a collection of Voxels which adds the energy of the blob. """
-    def __init__(self, a: Voxel, voxels : List[Voxel]) ->None:
-        super().__init__(voxels)
-        self.seed = a
+class Blob():
+    """A Blob is a collection of Hits with a seed and a radius. """
+    def __init__(self, a: Tuple[float], hits : List[BHit], radius : float) ->None:
```

Comment:
The name `seed` was decided last year in the sprint week of January 2018, if I remember correctly, when we reviewed the event model. I think that we decided to keep a generic name because it doesn't need to be the centre of a sphere (we can change the definition of blob at some point, as we already did, and not change the `Blob`class). However, it can be changed if we want to.
I have changed the type hint and the name of the argument for the seed.